src/core/api/index.js

```diff
@@ -55,7 +55,7 @@ export function callApi({ endpoint, schema, params = {}, auth = false, state = {
     .then((response) => (schema ? normalize(response, schema) : response));
 }
 
-export function search({ api, page, query }) {
+export function adminSearch({ api, page, query }) {
```

Comment:
Err, yeah I meant prefix when I said endpoint, I mistyped. But cool, that makes sense.